build/buildBuildImageBases.sh

```diff
@@ -16,56 +14,206 @@ declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
 
 # Load all variables
 source $REPO_DIR/build/__variables.sh
+source $REPO_DIR/build/__phpVersions.sh
 
 IMAGE_DIR_TO_BUILD=$1
-IMAGE_TAG="${BUILD_NUMBER:-latest}"
-BUILD_IMAGE_PREFIX="$__REPO_DIR/images/build"
+BUILD_DIR_PREFIX="$__REPO_DIR/images/build"
 ARTIFACTS_FILE="$BASE_IMAGES_ARTIFACTS_FILE_PREFIX/$IMAGE_DIR_TO_BUILD-buildimage-bases.txt"
 
 # Clean artifacts
 mkdir -p `dirname $ARTIFACTS_FILE`
 > $ARTIFACTS_FILE
 
+volumeHostDir="$ARTIFACTS_DIR/platformSdks"
+volumeContainerDir="/tmp/sdk"
+mkdir -p "$volumeHostDir"
+imageName="oryx/platformsdk"
+
+getSdkFromImage() {
+	local imageName="$1"
+
+	echo "Copying sdk file to host directory..."
+	echo
+	docker run \
+		-v $volumeHostDir:$volumeContainerDir \
+		$imageName \
+		bash -c "cp -f /tmp/compressedSdk/* /tmp/sdk"
+}
+
+blobDoesNotExist() {
```

Comment:
Name could be confusing - remove the negation maybe (i.e. `blobExists` or `doesBlobExist`)